Tests/Tests.swift

```diff
@@ -1115,14 +590,64 @@ class UnitTestSpec: QuickSpec {
                     .chainTransformation()
                     .rotation(rotation: Rotation.VALUE_90)
                     .create()
-                expect(actual).to(equal(String(format: "https://ik.imagekit.io/demo/medium_cafe_B1iTdD0C.jpg?ik-sdk-version=ios-%@&tr=h-300,w-300:rt-90", apiVersion)))
+                expect(actual).to(equal(String(format: "https://ik.imagekit.io/demo/medium_cafe_B1iTdD0C.jpg?tr=h-300,w-300:rt-90")))
+            }
+            it("With raw transforms string") {
+                let actual = ImageKit.shared
+                    .url(src: "https://ik.imagekit.io/demo/medium_cafe_B1iTdD0C.jpg")
+                    .raw(params: "h-300,w-300,rt-90")
+                    .create()
+                expect(actual).to(equal(String(format: "https://ik.imagekit.io/demo/medium_cafe_B1iTdD0C.jpg?tr=h-300,w-300,rt-90")))
             }
```

Comment:
add test for raw that includes the layer syntax

https://ik.imagekit.io/demo/img/plant.jpeg?tr=w-400:l-text,i-Hand%20with%20a%20green%20plant,co-264120,fs-30,lx-10,ly-20,l-end

you can use this one

Reply:
Done